Specifications/Compiler/com.apple.compilers.llvm.clang.1_0.migrator.xcspec

```diff
@@ -0,0 +1,41 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+(
+    {
+        Type = Compiler;
+        Identifier = com.apple.compilers.llvm.clang.1_0.migrator;
+        BasedOn = com.apple.compilers.llvm.clang.1_0;
+
+        InputFileTypes = (
+            "sourcecode.c.objc",
+            "sourcecode.cpp.objcpp",
+        );
+
+        SourceFileOption = "--migrate";
+
+        OutputDir = "$(TEMP_DIR)/Migrator";
+        OutputFileExtension = "remap";
```

Comment:
Like the static analyzer, this is pretty incomplete — could you either fill it out or (minimally) add a TODO comment?